docs/cog_guides/core.rst

```diff
@@ -0,0 +1,3393 @@
+Show the current help settings.
+
+.. Warning:: These setting may not be accurate if the default formatter is not in use.
+
+
+.. _core-command-helpset-tagline:
+
+"""""""""""""""
+helpset tagline
+"""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]helpset tagline [tagline]
+
+**Description**
+
+Set the tagline to be used.
+
+The maximum tagline length is 2048 characters.
+This setting only applies to embedded help. If no tagline is specified, the default will be used instead.
+
+Examples:
+    - ``[p]helpset tagline Thanks for using the bot!``
+    - ``[p]helpset tagline`` - Resets the tagline to the default.
+
+**Arguments:**
+
+- ``[tagline]`` The tagline to appear at the bottom of help embeds. Leave blank to reset.
+
+.. _core-command-helpset-usemenus:
+
+""""""""""""""""
+helpset usemenus
+""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]helpset usemenus [use_menus]
+
+**Description**
+
+Allows the help command to be sent as a paginated menu instead of separate
+messages.
+
+When enabled, ``[p]help`` will only show one page at a time and use reactions to navigate to other pages.
+
+This defaults to False.
+Using this without a setting will toggle.
+
+ Examples:
+    - ``[p]helpset usemenues False`` - Disables using menus on this server.
+    - ``[p]helpset usemenues`` - Toggles the value.
+
+**Arguments:**
+
+- ``[use_menus]`` Whether to use menus on this server. Leave blank to toggle.
+
+.. _core-command-helpset-usetick:
+
+"""""""""""""""
+helpset usetick
+"""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]helpset usetick [use_tick]
+
+**Description**
+
+This allows the help command message to be ticked if help is sent in a DM.
+
+Defaults to False.
+Using this without a setting will toggle.
+
+.. Note:: This is only used when the bot is not using menus.
+
+
+Examples:
+    - ``[p]helpset usetick False`` - Disables ticking in DMs.
+    - ``[p]helpset usetick`` - Toggles the value.
+
+**Arguments:**
+
+- ``[use_tick]`` Whether to tick the help command in DMs. Leave blank to toggle.
+
+.. _core-command-helpset-verifychecks:
+
+""""""""""""""""""""
+helpset verifychecks
+""""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]helpset verifychecks [verify]
+
+**Description**
+
+Sets if commands which can't be run in the current context should be filtered from help.
+
+Defaults to True.
+Using this without a setting will toggle.
+
+Examples:
+    - ``[p]helpset verifychecks False`` - Enables showing unusable commands.
+    - ``[p]helpset verifychecks`` - Toggles the value.
+
+**Arguments:**
+
+- ``[verify]`` Whether to hide unusable commands in help. Leave blank to toggle.
+
+.. _core-command-helpset-verifyexists:
+
+""""""""""""""""""""
+helpset verifyexists
+""""""""""""""""""""
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]helpset verifyexists [verify]
+
+**Description**
+
+This allows the bot to respond indicating the existence of a specific help topic even if the user can't use it.
```

Comment:
This was the existing description for the command. I will look into it further and get back to this one.

Reply:
I believe it is accurate as written. 

As an example:
With `verifyexists` = False, my bot will not respond to `[p]help cleanup duplicates` on the Red support server since it does not have manage messages permissions.
With `verifyexists` = True, my bot will respond to `[p]help cleanup duplicates` despite not being able to use it.

Reply:
This is now being handled in #4892